front/app/modules/commercial/insights/services/insightsInputs.ts

```diff
@@ -97,7 +96,6 @@ export async function addInsightsInputCategory(
   );
   streams.fetchAllWith({
     regexApiEndpoint: [inputsEndpointRegexp],
```

Comment:
This Is not introduced by this PR but still needs to be fixed :  It's too strict and doesn't refresh the queries that have filters or sort, we need to make it more flexible maybe with the partial version, or maybe changing the regex to accept params (To reproduce go in a category, change sort, add a category to one of the inputs via the sidemodal, change sort, your new category disappears, same with switching categories I think)

Reply:
Very good point! I learned the partial match existed recently, will use it here 👍